website/src/data/users.tsx

```diff
@@ -205,6 +205,15 @@ const Users: User[] = [
     source: 'https://github.com/ipavlic/apex-fp/tree/master/website',
     tags: ['opensource'],
   },
+  {
+    title: 'Astronomer',
+    description:
+      'Enterprise-grade framework for Apache Airflow. Production-ready Airflow environments with just a few clicks',
+    preview: require('./showcase/astronomer.png'),
```

Comment:
https://raw.githubusercontent.com/facebook/docusaurus/2ab5779dec7ed0ac33459e78b190c4946876185f/website/src/data/showcase/astronomer.png

this image does not have enough "height", we need a minimum aspect ratio for our showcase to render correctly, please reduce the width of your screen capture and add more height